packages/flutter/lib/src/widgets/focus_manager.dart

```diff
@@ -1082,6 +1135,9 @@ class FocusNode with DiagnosticableTreeMixin, ChangeNotifier {
   void debugFillProperties(DiagnosticPropertiesBuilder properties) {
     super.debugFillProperties(properties);
     properties.add(DiagnosticsProperty<BuildContext>('context', context, defaultValue: null));
+    if (this is! FocusScopeNode) {
```

Comment:
Oh, whoops, that was from before when the attribute was "overridesDescendants", and FocusScopeNodes always did that, so printing it was too verbose, since it was always true.  Now it doesn't make any sense for this to be different.

Removed the conditional.